processed 18 tasks

init:
A: object(0,0)

task 1 'publish'. lines 9-111:
created: object(1,0)
mutated: object(0,1)
gas summary: computation_cost: 1000000, storage_cost: 13452000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'run'. lines 113-113:
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3 'run'. lines 115-119:
mutated: object(0,0)
deleted: object(2,0)
gas summary: computation_cost: 1000000, storage_cost: 988000,  storage_rebate: 2573208, non_refundable_storage_fee: 25992

task 4 'run'. lines 121-121:
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 5 'run'. lines 123-123:
created: object(5,0), object(5,1)
mutated: object(0,0), object(4,0)
gas summary: computation_cost: 1000000, storage_cost: 6665200,  storage_rebate: 2573208, non_refundable_storage_fee: 25992

task 6 'run'. lines 125-129:
Error: Error checking transaction input objects: InvalidChildObjectArgument { child_id: object(5,0), parent_id: object(5,1) }

task 7 'run'. lines 131-131:
created: object(7,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2652400,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 8 'run'. lines 133-136:
Error: Transaction Effects Status: Invalid command argument at 0. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: TypeMismatch }, source: None, command: Some(1) } }

task 9 'run'. lines 138-138:
created: object(9,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2652400,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 10 'run'. lines 140-140:
created: object(10,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 11 'run'. lines 142-145:
Error: Transaction Effects Status: Invalid command argument at 1. The type of the value does not match the expected type
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 1, kind: TypeMismatch }, source: None, command: Some(0) } }

task 12 'run'. lines 147-147:
created: object(12,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 13 'run'. lines 149-152:
mutated: object(0,0)
deleted: object(12,0)
gas summary: computation_cost: 1000000, storage_cost: 988000,  storage_rebate: 2573208, non_refundable_storage_fee: 25992

task 14 'run'. lines 154-154:
created: object(14,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 15 'run'. lines 156-159:
Error: Transaction Effects Status: Invalid command argument at 0. Invalid usage of value. Mutably borrowed values require unique usage. Immutably borrowed values cannot be taken or borrowed mutably. Taken values cannot be used again.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidValueUsage }, source: None, command: Some(1) } }

task 16 'run'. lines 161-161:
created: object(16,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 2599200,  storage_rebate: 978120, non_refundable_storage_fee: 9880

task 17 'run'. lines 163-163:
Error: Transaction Effects Status: Invalid command argument at 0. Invalid usage of value. Mutably borrowed values require unique usage. Immutably borrowed values cannot be taken or borrowed mutably. Taken values cannot be used again.
Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: CommandArgumentError { arg_idx: 0, kind: InvalidValueUsage }, source: None, command: Some(1) } }
